### ISSUE TITLE
Event.scancode returns the wrong value for a comma keystroke

### ISSUE THREAD

**Issue by krwolcott** [godotengine/godot#23459]
** Godot Version 3.0.6**  (Steam Download, but reproducible in regular download.)
(Edit: This bug is also present in the current build of version 2.1.5)

** Windows 10 64-Bit**

**Issue description:**
When reading a comma, the scancode member of InputEventKey reports a colon.
Specifically, pressing the (edit) comma/less-than key on a US 101key keyboard produces an InputEventKey event with a scancode value of 58 and a unicode value of 44.

**Steps to reproduce:**
Create a new scene with a Control as root and a Label as a child.
Add a script to report the scancode and unicode.
Run the scene.
Press the comma/less-than key (on main keyboard between M and period).
scancode will be reported as 58 (colon) instead of 44 (comma).

**Minimal reproduction project:**
Control
Label
Label2
Label3

Control.gd
````
extends Control

var keys = {KEY_A: "A", KEY_B: "B", KEY_C: "C", KEY_COMMA: ",", KEY_COLON: ":"}

func _input(event):
    if event is InputEventKey:
        $Label.text = String(event.scancode) + ", " + String(unicode)
        $Label2.text = convert_scancode_to_string(event.scancode)    # Will show a colon
        $Label3.text = convert_scancode_to_string(event.unicode)    # Will show a comma

func convert_scancode_to_string(scancode):
    if keys.has(scancode):
        return keys[scancode]
    else:
        return "[unmatched scancode]"
````

**Added info**  Issue also affects the editor itself.  While trying to eliminate the possibility that I had made a newbie mistake, I tried creating a mapping in the Project Settings Input Map.  When attempting to bind a command to the comma key, the engine reports a colon instead.

**To reproduce:**
Open a project,
Go to the Project Settings window
Go to the Input Map page
Create a new action
Add a key bind to the action
Press the comma/less-than key to assign.

result, the dialog indicates that you pressed the colon key.

**Comment by PetePete1984** [comment]
Is that numpad comma or the default comma?

**Comment by krwolcott** [comment]
Default comma, next to M.

**Comment by bojidar-bg** [comment]
Duplicate of #6485, has been fixed by #21859 on master, will be cherrypicked for future 3.0.x versions.
